docs/tools/vdb_table/data/databricks.json

```diff
@@ -0,0 +1,188 @@
+    },
+    "facets": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "geo_search": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "multi_vec": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "sparse_vectors": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "bm25": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#how-does-mosaic-ai-vector-search-work",
+      "comment": "Okapi BM25"
+    },
+    "full_text": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "comment": ""
+    },
+    "embeddings_text": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": ""
+    },
+    "embeddings_image": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": "With self-managed Delta Sync"
+    },
+    "embeddings_structured": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": "With self-managed Delta Sync"
+    },
+    "rag": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/retrieval-augmented-generation.html#databricks-vector-search",
+      "comment": ""
+    },
+    "recsys": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": ""
+    },
+    "langchain": {
+      "support": "full",
+      "source_url": "https://python.langchain.com/v0.2/docs/integrations/vectorstores/databricks_vector_search/",
+      "comment": ""
+    },
+    "llamaindex": {
+      "support": "full",
+      "source_url": "https://docs.llamaindex.ai/en/stable/examples/vector_stores/DatabricksVectorSearchDemo/",
+      "comment": ""
+    },
+    "managed_cloud": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "comment": ""
+    },
+    "pricing": {
+      "value": "full",
+      "source_url": "https://www.databricks.com/product/pricing/vector-search",
+      "comment": "Includes free tier"
+    },
+    "in_process": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "multi_tenancy": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "comment": "via namespaces"
+    },
+    "disk_index": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html",
```

Comment:
link doesn't say anything about disk index

Reply:
Hi @kobindra, I understand the confusion. This is an important point for developers, as the scale of vector count that can be reached for a specific price point is fairly dependent on the possible storage medium of the vectors.

A lot of Vector DBs (proprietary and OSS) don't support storing indexes on Disk.
This reduces their scalability a fair bit.
In this case, you can just leave the source_url as it is. Should be good to go.

Reply:
@kobindra while I agree that the proprietary nature of the system cannot expose the details of the implementation, I think the documentation page should at least state that this is what's happening under the hood. This is one of the first things we (as developers) would like to understand when choosing a system to benchmark against. Would it be possible to add a couple of lines in the docs URL that's linked to mention this?

Reply:
@prrao87 Adding some clarifications in the docs is a great idea. I'll follow up on this with the team.